packages/adblocker/src/resources.ts

```diff
@@ -307,7 +317,7 @@ export default class Resources {
       dataUrl = `data:${contentType};base64,${btoaPolyfill(body)}`;
     }
 
-    return { body, contentType, dataUrl };
+    return { name, body, contentType, dataUrl };
```

Comment:
maybe instead of `name` we should call it a `filename` or `war`? (war may be too web extension specific)

Reply:
let stick with the `filename`

Reply:
pending with [`55d4046` (#4840)](https://github.com/ghostery/adblocker/pull/4840/commits/55d40461d53b7aca6a46e94e530672ae00dc6839)

Reply:
updated